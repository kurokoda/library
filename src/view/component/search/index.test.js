import React from "react";
import renderer from "react-test-renderer";

import Search from ".";

const props = {
  onClear: () => {},
  onSubmit: () => {},
};

it("renders correctly", () => {
  const tree = renderer.create(<Search {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});
