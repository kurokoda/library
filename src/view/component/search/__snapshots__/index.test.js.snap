// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <h2>
    Search For Your Books
  </h2>,
  <div
    className="library__search-container"
  >
    <input
      className="form-control"
      onChange={[Function]}
      onKeyDown={[Function]}
      placeholder="Enter a search term"
      type="text"
      value=""
    />
    <div
      className="btn-group"
      role="group"
    >
      <button
        aria-label={null}
        className="btn btn-primary"
        onClick={[Function]}
        type="button"
      >
        Search
      </button>
      <button
        aria-label={null}
        className="btn btn-danger"
        onClick={[Function]}
        type="button"
      >
        X
      </button>
    </div>
  </div>,
]
`;
